### PR TITLE
Add setAuto

### PR DESCRIPTION
**Pull request by liamcain**

### CODE REVIEW

**Review by dblarons — DISMISSED** [review]

**Comment by dblarons** [comment]
docs when you have a chance

File: src/BaseBuilder.js [hunk 1]
```diff
@@ -65,6 +65,20 @@ export default class BaseBuilder {
   }
 
   /**
+   * Set what property should be automatically created.
+   *
+   * Supported Options:
+   * - placeholders
+   * - labels
```
Comment (dblarons) [inline src/BaseBuilder.js]: Should `- none` also be a supported option?

https://github.com/gcanti/tcomb-form/blob/master/GUIDE.md#automatically-generated-placeholders

> Or auto: 'none' if you don't want neither labels nor placeholders

Comment (liamcain) [reply src/BaseBuilder.js]: yeah, good call

**Review by dblarons — APPROVED** [review]

**Review by dblarons — COMMENTED** [review]
Bump version

**Comment by dblarons** [comment]
nevermind, grouping versions (spoke in person)

**Pull request merged**